source_gen/lib/src/library.dart

```diff
@@ -98,10 +98,7 @@ class LibraryReader {
   /// * `asset`
   ///
   /// May throw [ArgumentError] if it is not possible to resolve a path.
-  Uri pathToUrl(dynamic toUrlOrString) {
```

Comment:
This was statically breaking.

This is worth doing, but we might have to wait for a major version bump, or add a `dart fix` fix for it.

Reply:
https://github.com/dart-lang/source_gen/pull/673

Reply:
annoying!